.github/workflows/deployment-development.yml

```diff
@@ -0,0 +1,54 @@
+# Deployment workflow to be used for Jira reporting
+name: Deploy alpha to Development
+
+on:
+  push:
+    branches: ["alpha"]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+jobs:
+  deploy-dev:
+    name: Deploy to Dev environment
+
+    # IMPORTANT: the workflow must have write access to deployments, otherwise the action will fail.
+    permissions:
+      deployments: write
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - uses: chrnorm/deployment-action@v2
+        name: Create GitHub deployment
+        id: deployment-dev
+        with:
+          token: '${{ github.token }}'
+          environment: development
+      # This step could be used in the future to notify or trigger another action
+      - name: Deploy my app
+        run: |
+          # add your deployment code here
+
+      - name: Update deployment status (success)
+        if: ${{ success() }}
+        uses: chrnorm/deployment-status@v2
+        with:
+          token: '${{ github.token }}'
+          state: 'success'
+          deployment-id: ${{ steps.deployment-dev.outputs.deployment_id }}
+
+      - name: Update deployment status (failure)
```

Comment:
Is this supposed to run if "Update deployment status (success)" fails?

Reply:
Maybe we could just merge both steps into one, and change the "state" value with an if condition

Reply:
The merge would over complicate the checking(discovered that after doing some research) would leave it simple like this and as recommended as well

Reply:
Ok